Document new rules for database migrations (#342)
peopledoc/procrastinate: CONTRIBUTING.rst

    $ tox -e generate-django-migrations

Backward-compatibility
^^^^^^^^^^^^^^^^^^^^^^

As a Procrastinate developer, the changes that you make to the Procrastinate database
schema must be compatible with the Python code of previous Procrastinate versions.

For example, let's say that the current Procrastinate database schema includes an SQL
function

.. code-block:: sql

    procrastinate_func(arg1 integer, arg2 text, arg3 timestamp)

that you want to change to

.. code-block:: sql

    procrastinate_func(arg1 integer, arg2 text)

> The naive way to do that is to edit the ``schema.sql`` file and just replace the old

Reviewer: ```suggestion
The straightforward way to do that would be to edit the ``schema.sql`` file and just replace the old
```
(that might make a line a bit too long, so you might need to balance the lines)
